src/Accelerators/NNPA/NNPAAccelerator.cpp

```diff
@@ -58,7 +58,13 @@ NNPAAccelerator::NNPAAccelerator() : Accelerator(Accelerator::Kind::NNPA) {
 
   acceleratorTargets.push_back(this);
   // Order is important! libRuntimeNNPA depends on libzdnn
-  addCompilerConfig(CCM_SHARED_LIB_DEPS, {"RuntimeNNPA", "zdnn"}, true);
+  std::vector<std::string> libs;
+  libs.push_back("RuntimeNNPA");
+  libs.push_back("zdnn");
+#ifdef ZDNNX_WITH_OMP
+  libs.push_back("gomp");
```

Comment:
why using Gnu OpenMP. It is never recommended to use two distinct OpenMP runtime at the same time. Please use libomp from LLVM

Reply:
It can be build in LLVM, and already get linked in onnx-mlir when libomp is built in LLVM. 

```
   -DLLVM_ENABLE_PROJECTS="mlir;clang;openmp" \
``` 
to build LLVM (think clang can be omitted).

Reply:
I changed to using ompruntime built by LLVM which is used by `--parallel` flag also. Thanks for the suggestion!